crates/mipsy_lib/src/inst/register.rs

```diff
@@ -40,16 +40,37 @@ pub enum Register {
 }
 
 pub const REGISTERS: [Register; 32] = [
-    Register::Zero, Register::At, 
-    Register::V0, Register::V1, 
-    Register::A0, Register::A1, Register::A2, Register::A3, 
-    Register::T0, Register::T1, Register::T2, Register::T3, 
-    Register::T4, Register::T5, Register::T6, Register::T7, 
-    Register::S0, Register::S1, Register::S2, Register::S3, 
-    Register::S4, Register::S5, Register::S6, Register::S7, 
-    Register::T8, Register::T9, 
-    Register::K0, Register::K1, 
-    Register::Gp, Register::Sp, Register::Fp, 
+    Register::Zero,
+    Register::At,
+    Register::V0,
+    Register::V1,
+    Register::A0,
+    Register::A1,
+    Register::A2,
+    Register::A3,
+    Register::T0,
+    Register::T1,
+    Register::T2,
+    Register::T3,
+    Register::T4,
+    Register::T5,
+    Register::T6,
+    Register::T7,
+    Register::S0,
+    Register::S1,
+    Register::S2,
+    Register::S3,
+    Register::S4,
+    Register::S5,
+    Register::S6,
+    Register::S7,
+    Register::T8,
+    Register::T9,
+    Register::K0,
+    Register::K1,
+    Register::Gp,
+    Register::Sp,
+    Register::Fp,
```

Comment:
I would rather this be left in the more compact (and organised) original form, you can add `#[rustfmt::skip]` to line 41 to preserve the formatting when running `rustfmt` on everything else